v22.1/api-support-policy.md

```diff
@@ -0,0 +1,89 @@
+| [SQL syntax](sql-feature-support.html)                                  | Mixed    | Versioned concurrently with CockroachDB.                                               | Best-effort policy to add and not remove SQL syntax. All `SHOW` statements are unstable, as described in the following row.                                  | All products                                                      |
+| `SHOW` SQL statements                                                   | Unstable | Versioned concurrently with CockroachDB.                                               | This includes all documented SQL `SHOW` statements, which display unstable output.                                                                           | All products                                                      |
+| [`crdb_internal` system catalog](crdb-internal.html)                    | Mixed    | Versioned concurrently with CockroachDB.                                               | A subset of the `crdb_internal` system catalog is stable, as defined [here](crdb-internal.html#tables).                                                      | All products                                                      |
+| [`information_schema` system catalog](information-schema.html)          | Stable   | Versioned concurrently with CockroachDB.                                               |                                                                                                                                                              | All products                                                      |
+| [`pg_catalog` system catalog](pg-catalog.html)                          | Stable   | Versioned concurrently with CockroachDB.                                               |                                                                                                                                                              | All products                                                      |
+| [`pg_extension` system catalog](pg-extension.html)                      | Stable   | Versioned concurrently with CockroachDB.                                               |                                                                                                                                                              | All products                                                      |
+| [Built-in functions](functions-and-operators.html#built-in-functions)   | Mixed    | Versioned concurrently with CockroachDB.                                               | Any built-in functions prefixed with `crdb_internal` are unstable.                                                                                           | All products                                                      |
```

Comment:
I think we should be saying that `crdb_internal` functions are _reserved_ rather than unstable